src/components/EditCampaign.tsx

```diff
@@ -86,8 +86,7 @@ class EditCampaign extends React.Component<Props, State> {
 
   public render () {
     if (this.state.redirect) {
-      redirect(`/campaigns`);
-      return null;
+      return <Navigate replace={true} to={`/campaigns`}/>;
```

Comment:
No variable in the 'to', so do you need the template literal, could it just be a string as in other components?